tech-specs/src/explore/design/pos.md

```diff
@@ -0,0 +1,258 @@
+# Proof of Stake (PoS) system
+
+## Epoch
+
+An epoch is a range of blocks, whose length is set by the `epoch_duration` [parameter](#system-parameters). Epochs are identified by consecutive integers starting at 0. For each epoch, we set the [relevant data](#epoched-data) in advance by the `pipeline_length` [parameter](#system-parameters).
+
+An epoch starts at the beginning of block height `n` and end at the end of block height `n + epoch_duration - 1`. Then the following epoch starts at the beginning of block height `epoch_duration`, etc.
+
+### Epoched data
+
+The data relevant to the PoS system in the ledger's state are epoched. Each data can be uniquely identified. These are:
```

Comment:
What does "epoched" mean? (presumably here "fixed for a particular epoch")

Reply:
I've reworded this to start the paragraph with `Epoched data are data associated with a specific epoch that are set in advance.`. Then there's more details in the following paragraph.